app/build.gradle.kts

```diff
@@ -1,281 +1,275 @@
+  implementation("androidx.constraintlayout:constraintlayout:2.1.4")
+  implementation("androidx.navigation:navigation-fragment-ktx:2.7.5")
+  implementation("androidx.navigation:navigation-ui-ktx:2.7.5")
+  implementation(platform("androidx.compose:compose-bom:2023.08.00"))
+  implementation("androidx.compose.ui:ui-graphics")
+  implementation("com.google.android.gms:play-services-auth:20.6.0")
+
+  implementation("com.google.firebase:firebase-auth-ktx:22.3.0")
+  implementation("com.google.firebase:firebase-database-ktx:20.3.0")
+  implementation("com.google.firebase:firebase-auth:22.3.1")
+  testImplementation("junit:junit:4.13.2")
+  androidTestImplementation("androidx.test.ext:junit:1.1.5")
+  androidTestImplementation("androidx.test.espresso:espresso-core:3.5.1")
+  androidTestImplementation("androidx.compose.ui:ui-test-junit4:1.4.0")
+  androidTestImplementation(platform("androidx.compose:compose-bom:2023.08.00"))
+  debugImplementation("androidx.compose.ui:ui-tooling:1.4.0")
+  debugImplementation("androidx.compose.ui:ui-test-manifest:1.4.0")
+
+  androidTestImplementation("com.kaspersky.android-components:kaspresso:1.4.3")
+  // Allure support
+  androidTestImplementation("com.kaspersky.android-components:kaspresso-allure-support:1.4.3")
+  // Jetpack Compose support
+  androidTestImplementation("com.kaspersky.android-components:kaspresso-compose-support:1.4.1")
+
+  implementation("androidx.fragment:fragment:1.5.5")
+
+  implementation("com.squareup.okhttp3:okhttp:3.10.0")
+
+  testImplementation("org.mockito:mockito-core:3.11.2")
+  testImplementation("org.mockito:mockito-inline:2.13.0")
+  implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.6.2")
+
+  implementation(platform("com.google.firebase:firebase-bom:32.7.2"))
+  implementation("com.firebaseui:firebase-ui-auth:7.2.0")
+  implementation("com.google.firebase:firebase-database-ktx:20.3.0")
+  implementation("com.google.firebase:firebase-firestore:24.10.0")
+  implementation("com.google.android.play:core-ktx:1.7.0")
+  implementation("com.google.firebase:firebase-auth-ktx:22.3.1")
+  implementation("com.google.android.gms:play-services-auth:19.2.0")
+
+  implementation("androidx.navigation:navigation-compose:2.6.0-rc01")
+
+  // live data
+  implementation("androidx.compose.runtime:runtime-livedata:1.1.0")
+
+  // Async Image
+  implementation("io.coil-kt:coil-compose:2.6.0")
+
+  val nav_version = "2.7.7"
+
+  // Kotlin
+  implementation("androidx.navigation:navigation-fragment-ktx:$nav_version")
+  implementation("androidx.navigation:navigation-ui-ktx:$nav_version")
+
+  // Testing Navigation
+  androidTestImplementation("androidx.navigation:navigation-testing:$nav_version")
+
+  // OkHTTP
+  implementation("com.squareup.okhttp3:okhttp:4.12.0")
+
+  // tests
+  // Dependency for using Intents in instrumented tests
+  androidTestImplementation("androidx.test.espresso:espresso-intents:3.5.1")
+
+  // Dependencies for using MockK in instrumented tests
```

Comment:
Strange that git sees so many deletions when the code is the same.

This is the important change you made! Thanks for solving the Proxy Error issue thet we had Jeremy! Much appreciated.